platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/gui/GuiPlatformAccessor.java

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.gui;
+
+import com.swirlds.platform.Consensus;
+import com.swirlds.platform.components.state.StateManagementComponent;
+import com.swirlds.platform.gossip.shadowgraph.ShadowGraph;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import edu.umd.cs.findbugs.annotations.Nullable;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * Provides a way to access private platform objects from the GUI. Suboptimal, but necessary to preserve the current UI
+ * architecture if we don't want to allow public access to these objects.
+ *
+ * @deprecated this class will eventually be removed
+ */
+@Deprecated(forRemoval = true)
+public final class GuiPlatformAccessor {
+
+    private final Map<Long, String> aboutStrings = new ConcurrentHashMap<>();
+    private final Map<Long, String> platformNames = new ConcurrentHashMap<>();
+    private final Map<Long, byte[]> swirldIds = new ConcurrentHashMap<>();
+    private final Map<Long, Integer> instanceNumbers = new ConcurrentHashMap<>();
+    private final Map<Long, ShadowGraph> shadowGraphs = new ConcurrentHashMap<>();
+    private final Map<Long, StateManagementComponent> stateManagementComponents = new ConcurrentHashMap<>();
+    private final Map<Long, AtomicReference<Consensus>> consensusReferences = new ConcurrentHashMap<>();
+
+    private static final GuiPlatformAccessor INSTANCE = new GuiPlatformAccessor();
+
+    /**
+     * Get the static instance of the GuiPlatformAccessor.
+     *
+     * @return the static instance of the GuiPlatformAccessor
+     * @deprecated this method is deprecated and will be removed in a future release
+     */
+    @Deprecated(forRemoval = true)
```

Comment:
adding it to the class should be good enough

Reply:
removed all deprecated tags except for the one on the class definition